src/main/java/org/broadinstitute/hellbender/tools/spark/sv/SVUtils.java

```diff
@@ -124,4 +126,11 @@ public static int hashMapCapacity( final int nElements )
         while ( itr.hasNext() ) { result += 1; itr.next(); }
         return result;
     }
+
+    /**
+     * Provides a stream collector that will collect items into an array list with a given initial capacity.
+     */
+    static Collector<AlignmentRegion, ?, ArrayList<AlignmentRegion>> arrayListCollector(final int size) {
```

Comment:
Maybe genericize like this:
    static <T> Collector<T, ?, ArrayList<T>> arrayListCollector(final int size) {